SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -283,6 +284,11 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	virtual int64 GetActorEntityId(const AActor& Actor) const override;
 
+	void CheckUnauthorisedDataChanges(Worker_EntityId_Key EntityId, AActor* Actor);
```

Comment:
all these paramers could be const and/or ref.

Reply:
Made all parameters const.